plugins/modules/dconf.py

```diff
@@ -122,12 +129,20 @@
     key: "/org/cinnamon/desktop-effects"
     value: "false"
     state: present
+
+- name: Load terminal profile in Gnome
+  community.general.dconf:
+    key: "/org/gnome/terminal/legacy/profiles/:"
+    path: "/tmp/solarized_dark.dump"
+    state: present
 """
 
 
 import os
 import sys
 
+from configparser import ConfigParser
```

Comment:
This only works in Python 3. For Python 2, you need to use six:
```suggestion
from ansible.module_utils.six.moves.configparser import ConfigParser
```